src/main/java/uk/gov/hmcts/dts/fact/util/Utils.java

```diff
@@ -52,4 +57,12 @@ public static boolean isNorthernIrishPostcode(final String postcode) {
     public static String upperCaseAndStripAllSpaces(final String input) {
         return input.replaceAll("\\s+","").toUpperCase(Locale.getDefault());
     }
+
+    public static List<String> constructAddressLines(final String address) {
+        return address == null
```

Comment:
Anyway for us to not use nulls if possible?

Reply:
Yeah this is an existing method I moved from somewhere else. I will change it to use Stringutils.isBlank() for this one.